src/collective/upload/config.py

```diff
@@ -4,7 +4,7 @@
 
 IMAGE_MIMETYPES = ['image/jpeg', 'image/gif', 'image/png']
 # XXX: we don't want to put limits here; that should be user configurable
-UPLOAD_EXTENSIONS = u'gif, jpeg, jpg, png'
+UPLOAD_EXTENSIONS = u'gif|jpeg|jpg|png'
```

Comment:
Description of the field [upload_extensions](https://github.com/collective/collective.upload/blob/update-jquery-fileupload/src/collective/upload/interfaces.py#L32-L35) must be updated. Translations too.